src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
```

Comment:
Where is the getter for this private variable?
Consider adding the getter to the interface ReadOnlyPerson if you want to give access to that as well.